musit-models/src/main/scala/no/uio/musit/models/Module.scala

```diff
@@ -0,0 +1,25 @@
+package no.uio.musit.models
+
+import play.api.libs.json.{JsNumber, Reads, Writes, __}
+
+sealed trait Module {
+  val id: Int
+}
+
+case object StorageFacility extends Module {
+  val id = 1
+}
+case object LoanProcess extends Module {
+  val id = 2
+}
+
```

Comment:
The "LoanProcess" type was meant as irony. But it needs a little bit of context ;) But yes, it should be Loan.